frontend/src/utils/layers-utils.tsx

```diff
@@ -94,6 +95,14 @@ const useLayers = () => {
     return bbox(boundaryLayerData.data) as Extent; // we get extents of admin boundaries to give to the api.
   }, [boundaryLayerData]);
 
+  // TODO - investigate if this is not too time consuming for larger countries.
```

Comment:
@laurentS we need to investigate if this is the right place to do this calculation and what the performance impact is on a larger deployment like RBD.

Reply:
Well it crashed for Cambodia so that's an answer 😅

Reply:
Maybe we should simply run a script to save this boundary once per country

Reply:
To be sure I'm not misreading the code, this calculates a single boundary polygon that includes the entire country's (or multicountries' for RBD) area. I can't really see the value in recalculating this on every app run. By definition, it won't change more often than the underlying boundary files. I'd vote for precomputing this, either at deployment time (or when the dev version starts), or even add the output in git along with the boundary definitions (produced using the code you wrote here). As a side note, having that one big polygon available could also speed up `boundaryLayerDataAreOutsideOfBoundaryBBox` which is called on every click on the map (to check if we clicked inside the country).